string-valid.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Takes a string, returns true of false
+ * https://rune-taurus-c3a.notion.site/Mock-Interview-dd069edb04204237af06fd7467f509aa
+ * @param {string} str
+ * @returns boolean
+ */
+export function insertingString(str) {
+  if (str == "") {
+    return "abc";
```

Comment:
this should be probably `true` rather than a string, would "abc" as a result of this function would even mean?